Block/AdminListBlockService.php

```diff
@@ -81,8 +81,13 @@ public function setDefaultSettings(OptionsResolverInterface $resolver)
             'groups' => false
         ));
 
-        $resolver->setAllowedTypes(array(
-            'groups' => array('bool', 'array')
-        ));
+        $setAllowedTypesMethod = new \ReflectionMethod($resolver, 'setAllowedTypes');
```

Comment:
Can you use ``version_compare`` with ``Kernel::VERSION`` to avoid using reflection ?